src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BlockListener.java

```diff
@@ -63,18 +63,16 @@ public void onBlockPlaceExisting(BlockPlaceEvent e) {
          */
         Block block = e.getBlock();
 
-        if (e.getBlockReplacedState().getType().isAir()) {
+        if (e.getBlockReplacedState().getType().isAir() && !SlimefunPlugin.getTickerTask().isPendingDeletion(block.getLocation())) {
             SlimefunItem sfItem = BlockStorage.check(block);
-
             if (sfItem != null) {
-                /* Temp fix for #2636
                 for (ItemStack item : sfItem.getDrops()) {
                     if (item != null && !item.getType().isAir()) {
                         block.getWorld().dropItemNaturally(block.getLocation(), item);
                     }
                 }
-                 */
                 BlockStorage.clearBlockInfo(block);
+                e.setCancelled(true);
```

Comment:
Why is it cancelled here?

Reply:
While testing, I found out, that sometimes placed blocks lost their data (like Charging Bench becoming just crafting table)